src/System.CommandLine.Suggest/SuggestionDispatcher.cs

```diff
@@ -74,9 +74,8 @@ public SuggestionDispatcher(ISuggestionRegistration suggestionRegistration, ISug
                 RegisterCommand,
                 CompleteScriptCommand
             };
-
+            root.TreatUnmatchedTokensAsErrors = false;
```

Comment:
This looks like it will be a behavioral change. We shouldn't alter the parser configuration here.